.coveragerc

```diff
@@ -0,0 +1,14 @@
+[run]
+branch = True
+source = zc.form
+
+[report]
+precision = 2
+
+[html]
+directory = htmlcov
+
+[paths]
+source =
+   src/zc/form
```

Comment:
This is not necessary, I think.

`[paths]` are used by `coverage combine` when you want to tell coverage that the same code can be found in several different locations.

I don't think a path list with only one item would ever be useful.

Reply:
Okay, convinced.